src/bidiMapper/domains/network/NetworkStorage.ts

```diff
@@ -135,6 +130,14 @@ export class NetworkStorage {
     };
   }
 
+  get(id: Network.Request) {
+    return this.#requestMap.get(id);
+  }
+
+  delete(id: Network.Request) {
+    this.#requestMap.delete(id);
+  }
```

Comment:
Not a fan of these method names, because there are 3 maps in this class. It's not obvious which one it refers to.

Reply:
Added a Specific Request